avidemux_sk.ts

```diff
@@ -1999,47 +1999,47 @@ Vypustiť časové údaje?</translation>
     <message>
         <location line="+6"/>
         <source>Pictures</source>
-        <translation type="unfinished"></translation>
+        <translation>Obrázky</translation>
     </message>
     <message>
         <location line="+1"/>
         <source>_Load sequentially named pictures in reverse order</source>
-        <translation type="unfinished"></translation>
+        <translation>_Načítať sekvenciu pomenovaných obrázkov v opačnom poradí</translation>
     </message>
     <message>
         <location line="+2"/>
         <source>23.976 (Film)</source>
-        <translation type="unfinished"></translation>
+        <translation>23.976 (Film)</translation>
```

Comment:
According to `LANG=sk_SK locale decimal_point` the decimal separator in Slovak is a comma. I absolutely don't insist if there are good reasons to deviate, however.

Reply:
> According to `LANG=sk_SK locale decimal_point` the decimal separator in Slovak is a comma. I absolutely don't insist if there are good reasons to deviate, however.

While comma is indeed the official the official decimal separator a "dot" is quiet commonly seen in technical terms. Consider a list of frame rates just using commas: "23,976, 25,00, 30,00".

Reply:
> Consider a list of frame rates just using commas: "23,976, 25,00, 30,00".

Well, we don't have this problem here, but as there is no danger of mixing up thousands and decimal separator in Slovak, keeping dot is fine with me.